fe/fe-core/src/main/java/com/starrocks/sql/ast/PartitionDesc.java

```diff
@@ -46,7 +46,10 @@ public String toSql() {
         throw new NotImplementedException();
     }
 
-    public PartitionInfo toPartitionInfo(List<Column> columns, Map<String, Long> partitionNameToId, boolean isTemp)
+    // Currently, RANGE is used for materialized view ExpressionRangePartitionInfo, which is isExprPartition=false,
+    // and EXPR_RANGE is used for ordinary table ExpressionRangePartitionInfo, which is isExprPartition=true
+    public PartitionInfo toPartitionInfo(List<Column> columns, Map<String, Long> partitionNameToId,
```

Comment:
you can. add another function public PartitionInfo toPartitionInfo(List<Column> columns, Map<String, Long> partitionNameToId, boolean isTemp), which call toPartitionInfo with isExprPartition to false?